vault/token_store.go

```diff
@@ -747,6 +747,10 @@ func (ts *TokenStore) tokenStoreAccessorList(ctx context.Context, req *logical.R
 			resp.AddWarning(fmt.Sprintf("Found an accessor entry that could not be successfully decoded; associated error is %q", err.Error()))
 			continue
 		}
+		if aEntry == nil {
+			resp.AddWarning("Found an accessor entry that could not be successfully decoded: invalid accessor")
```

Comment:
I don't think we need this warning.  There's already the one below that occurs when "entry missing a token".  The warning about "can't successfully decode", which you're trying to maintain here, was I think inappropriate for the case you're changing, where there's not actually an accessorEntry in storage.  In between our list and our lookup the token may have been deleted, which is perfectly normal and doesn't warrant a warning IMO.